doc/modules/spf.md

```diff
@@ -43,14 +43,22 @@ spf {
 }
 ~~~
 
-Also, you can specify the IP addresses of external relays for checking the SPF Policy of the real sender.
+Also, you can specify the IP addresses of external relays for checking the SPF Policy of the real sender. You can do it by adding IP addresses to conf file (necessarily use an array) or by adding to `map`.
```

Comment:
>> You can do it by adding an IP address (or multiple addresses) to the configuration file directly or using an external `map`. Should you place the IP address or addresses in the configuration file directly please mention that in this case you need to use an array of string representing the desired addresses even if you add a single IP address, e.g. `['192.168.1.1']`. You can use networks in CIDR notation instead of IP addresses as well.